README.md

```diff
@@ -57,4 +57,40 @@ Labrador retriever: 7.1%
 tennis ball: 1.3%
 clumber: 0.9%
 Brittany spaniel: 0.7%
-```
\ No newline at end of file
+```
+
+### Run Secure Prediction with Encryption
+
+You can encrypt data before sending it to Cape to be processed in your function. You can encrypt
+to yourself or you can encrypt for another person. If you encrypt for yourself only you can decrypt the
+data and if you encrypt for another person only they can decrypt it.
+
+There are three examples for testing this out:
+
+Returns the encrypted string for you.
+
+```
+$ python encrypt.py
+Encrypted: cape:KTTGfoNTQu....
+```
+
+Returns the encrypted string for the capedocs user.
```

Comment:
Maybe we should mention what is this capedocs user?

Reply:
Done thanks Yann!